internal/core/converter/tensorflow/tensorflow_test.go

```diff
@@ -53,31 +52,74 @@ func TestNew(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           opts: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           opts: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "return (t, nil): default options",
+			args: args{
+				opts: nil,
+			},
+			want: want{
+				want: &tensorflow{
+					graph:   nil,
+					session: (&tf.SavedModel{}).Session,
+				},
+				err: nil,
+			},
+			checkFunc: defaultCheckFunc,
+			beforeFunc: func(args args) {
+				defaultOpts = []Option{}
+				loadFunc = func(s string, ss []string, o *SessionOptions) (*tf.SavedModel, error) {
+					return &tf.SavedModel{}, nil
+				}
+			},
+		},
+		{
+			name: "return (t, nil): args options",
+			args: args{
+				opts: []Option{
+					WithSessionTarget("test"),
+					WithSessionConfig([]byte{}),
+					WithNdim(1),
+				},
+			},
+			want: want{
+				want: &tensorflow{
+					sessionTarget: "test",
+					sessionConfig: []byte{},
+					options: &tf.SessionOptions{
+						Target: "test",
+						Config: []byte{},
+					},
+					graph:   nil,
+					session: (&tf.SavedModel{}).Session,
+					ndim:    1,
+				},
+				err: nil,
+			},
+			checkFunc: defaultCheckFunc,
```

Comment:
you can delete this line, because this function was set automatically in the  `tt.Run`.